### DISCUSSION TITLE
Promote to maintainer

### DISCUSSION THREAD

**Discussion by rrey** [ansible-collections/community.grafana#385] [General]
Hello @Nemental,

Would you be interested in becoming an official maintainer of the collection ?
You are doing all the work for the past month and it sounds legitimate that you are acknowledged as maintainer.

I guess that could also be the occasion to step back for me as I did not participate in project for a while.

WDYT ?

**Comment by Nemental** [comment]
Hey @rrey,
I'm very happy about that! Thanks for your trust and the suggestion to make me a maintainer. I really enjoy working on the collection.

**Comment by Nemental** [comment]
Hello @rrey 
Is the topic still relevant? I'm back from my holiday and will continue working on the project.

**Reply by rrey** [reply]
We can close the discussion I guess.
I forgot to come back to this but I'll elevate your privileges on the repository.

After a quick check with Ansible Community it seems there isn't anything required from their side to do that.
Thanks again for all the work @Nemental !

**Reply by Nemental** [reply]
Thank you! I really enjoy the work on this repository and I look forward to fix some issues and add new features ;) happy maintaining!

**Comment by rrey** [comment]
You now have "maintain" role in the repo @Nemental !
Congratulations ! 🎉 and thanks a lot !

**Reply by Nemental** [reply]
Great, thank you for your trust!
